src/SFA.DAS.Payments.ServiceFabric.Core/StatelessServiceBusBatchCommunicationListener.cs

```diff
@@ -0,0 +1,466 @@
+﻿using System;
```

Comment:
Quite a bit of commented out code in this file, can it be removed?

Reply:
Commented out code removed.